src/PrestaShopBundle/Form/Admin/Sell/CatalogPriceRule/CatalogPriceRuleType.php

```diff
@@ -0,0 +1,238 @@
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $dateTimeFormat = 'YYYY-MM-DD HH:mm:ss';
+
+        $builder
+            ->add('name', TextType::class, [
+                'constraints' => [
+                    new CleanHtml(),
+                ],
+            ])
+            ->add('id_currency', ChoiceType::class, [
+                'required' => false,
+                'placeholder' => false,
+                'choices' => $this->getModifiedCurrencyChoices(),
+            ])
+            ->add('id_country', ChoiceType::class, [
+                'required' => false,
+                'placeholder' => false,
+                'choices' => $this->getModifiedCountryChoices(),
+            ])
+            ->add('id_group', ChoiceType::class, [
+                'required' => false,
+                'placeholder' => false,
+                'choices' => $this->getModifiedGroupChoices(),
+            ])
+            ->add('from_quantity', TextType::class)
+            ->add('price', NumberType::class, [
+                'required' => false,
+                'scale' => 6,
+                'constraints' => [
+                    new GreaterThanOrEqual([
+                        'value' => 0,
+                        'message' => $this->translator->trans(
+                            '%s is invalid.',
+                            [],
+                            'Admin.Notifications.Error'
+                        ),
+                    ]),
+                ],
+            ])
+            ->add('leave_initial_price', CheckboxType::class, [
+                'required' => false,
+            ])
+            ->add('from', DatePickerType::class, [
+                'required' => false,
+                'date_format' => $dateTimeFormat,
+            ])
+            ->add('to', DatePickerType::class, [
+                'required' => false,
+                'date_format' => $dateTimeFormat,
+            ])
+            ->add('include_tax', ChoiceType::class, [
+                'placeholder' => false,
+                'required' => false,
+                'choices' => $this->taxInclusionChoices,
+            ])
+            ->add('reduction_type', ChoiceType::class, [
+                'placeholder' => false,
+                'required' => false,
+                'choices' => $this->priceReductionTypeChoices,
+            ])
+            ->add('reduction', NumberType::class, [
+                'scale' => 6,
+                'constraints' => [
+                    new GreaterThanOrEqual([
+                        'value' => 0,
+                        'message' => $this->translator->trans(
+                            '%s is invalid.',
+                            [],
+                            'Admin.Notifications.Error'
+                        ),
+                    ]),
+                ],
+            ])
+        ;
+
+        if ($this->isMultishopEnabled) {
+            $builder->add('id_shop', ChoiceType::class, [
```

Comment:
@matks, I see that shop field is not in the specs doc. Is it missed by mistake (I guess, yes, otherwise we don't know which shop to target), or it really shouldn't be there?

Reply:
ping @marionf again 😄
The specifications doc does mention the "shop" input for multistore, is that expected or a small mistake ?